web/src/components/Pretreatment.vue

```diff
@@ -13,6 +14,7 @@ export default {
   data() {
     return {
       datasets: this.$store.state.datasets,
+      analyses,
```

Comment:
Related to my last question, it seems like we're repeating some data storage in different places.

Reply:
I believe this is just making some constant values available to the template.  This doesn't need to be a reactive object and the template wont change its value, so it could just  as well be a functional component we factor out, but IMO that isn't necessary yet.